pkg/steps/multi_stage.go

```diff
@@ -240,18 +241,11 @@ func (s *multiStageTestStep) setupRBAC() error {
 		RoleRef:    rbacapi.RoleRef{Kind: "Role", Name: s.name},
 		Subjects:   subj,
 	}
-	check := func(err error) bool {
-		return err == nil || kerrors.IsAlreadyExists(err)
-	}
-	if err := s.client.Create(context.TODO(), sa); !check(err) {
-		return err
-	}
-	if err := s.client.Create(context.TODO(), role); !check(err) {
-		return err
-	}
-	if err := s.client.Create(context.TODO(), binding); !check(err) {
+
+	if err := util.CreateRBACs(context.TODO(), sa, role, binding, s.client, 1*time.Minute); err != nil {
```

Comment:
Why did you move that to a different package when its only used here? And its kind of confusing to have a `CreateRBAC` func that requires a ServiceAccount as argument. Also, doesn't the ServiceAccount poll code belong into `InitializeNamespace`?

Reply:
Ah, its also used in CreateRelease. Still, I think the part that needs factoring out is the SA creation + Polling, combining it with the RBAC just makes it less re-usable and more confusing

Reply:
There are currently two places that we are creating rbacs. Multi-stage and create_release. I moved it to `util` package because it makes sense to have this kind of functionality in a neutral place. In `initializeNamespace` for some reason, we don't create any related rbacs.

Reply:
yea. Also, they are creating different serviceaccounts. The whole concept here needs some refactoring for sure.